docs/source/azure/gettingstarted.rst

```diff
@@ -16,8 +16,30 @@ We recommend using Python 3.6 or higher.
 
 The Azure provider is an additional package which is installed in addition to c7n.
 
-Option 1: Install latest from the repository
-"""""""""""""""""""""""""""""""""""""""""""""
+Option 1: Install released packages to virtual Python environment
```

Comment:
If we decide to add this back in do we want to include the virtual env/activate?  Experience has been many users can't even get past that part.

We do have a new option that doesn't even require python coming as soon as I can write up the docs for it, probably next week.